LayoutTests/imported/w3c/web-platform-tests/css/css-grid/masonry/tentative/masonry-grid-template-columns-computed-withcontent.html

```diff
@@ -77,7 +77,7 @@
 // <auto-repeat> = repeat( [ auto-fill | auto-fit ] , [ <line-names>? <fixed-size> ]+ <line-names>? )
 test_computed_value("grid-template-columns", 'repeat(auto-fill, 200px)', '200px');
 test_computed_value("grid-template-columns", 'repeat(auto-fit, [one] 20%)',
-                    '[one] 60px [one] 0px [one] 0px [one] 0px [one] 0px');
+                    '[one] 60px [one] 60px [one] 60px [one] 60px [one] 60px');
```

Comment:
Want some feedback on this. We are supposed to treat it as an auto-fill, so this seems to make sense to me.

Reply:
I agree that this is a correct change to make, the tracks have a definite size and aren't being collapsed (since it's auto-fill, not auto-fit).

You'll need to upstream this change to WPT once it merges.